docs-ref-conceptual/node-azure-tools.md

```diff
@@ -13,7 +13,7 @@ ms.author: cshoe
 ---
 
 # Azure tools for Node.js developers
-We recommend the following cross-platform tools, which allow you to develop using Node.js on Azure.
+The following tools are reccomended for developing with Azure on Node.js.
```

Comment:
Type on "reccomended"

Reply:
Also recommend changing sentence from "We recommend the following cross-platform tools, which allow you to develop using Node.js on Azure." to "We recommend the following cross-platform tools that allow you to develop using Node.js on Azure."

Reply:
Thanks for the comments and spotting that typo 😃. I was trying to remove the pronoun, but I am guessing your suggestion is more in-line with what other docs have?